### PR TITLE
FW-6329, FW-6336 replace loading with loadorerror part 1

### PR DESCRIPTION
**Pull request by gmcauliffe**
### Description of Changes

- replace some instances of Loading with LoadOrError
- remove duplication in infinite scroll hooks
- fix broken video thumbnails in video selector modal
- reduce renaming of query return properties e.g. response?.data -> media

### Checklist

- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/components/SelectorVisualMediaGrid/SelectorVisualMediaGridPresentation.js [hunk 1]
```diff
@@ -36,7 +35,7 @@ function SelectorVisualMediaGridPresentation({
                   const mediaSrc = getMediaPath({
                     type: IMAGE,
                     mediaObject,
-                    size: THUMBNAIL,
```
Comment (bronson-c) [inline src/components/SelectorVisualMediaGrid/SelectorVisualMediaGridPresentation.js:39]: Is this change of thumbnails to small images to fix the broken video thumbnails?

File: src/components/Dictionary/DictionaryContainer.js [hunk 2]
```diff
@@ -4,40 +4,25 @@ import PropTypes from 'prop-types'
...
-    loadRef,
-    count,
-  } = DictionaryData({ searchType, kids })
+  const { infiniteQueryResponse, labels, sitename } = DictionaryData({
+    searchType,
+    kids,
+  })
   return (
-    <Loading.Container isLoading={isLoading}>
+    <>
```
Comment (bronson-c) [inline src/components/Dictionary/DictionaryContainer.js:16]: Just curious, are these empty blocks necessary? I noticed them in a lot of the container files.

**Review by bronson-c — APPROVED** [review]
LGTM! I just had a few questions based on code I noticed but they should not block the merge

File: src/components/Dictionary/DictionaryContainer.js (hunk 2 above)
Comment (gmcauliffe) [reply src/components/Dictionary/DictionaryContainer.js:16]: The empty blocks are react Fragments. JSX elements require a single parent element, and using a div can impact styling so sometimes a fragment is used instead.

**Pull request merged**